src/components/Header/AuxiliaryDropdown.tsx

```diff
@@ -23,6 +36,11 @@ export const AuxiliaryDropdown = () => {
       href: `${uiURL}${legacyRoutes.projects}`,
       text: "Projects",
     },
+    {
```

Comment:
Since this is tied to the mainline commits project it might make sense to toggle this with the spruce-beta flag. The cookie only gets set from the waterfall page so its possible the user won't find this link useful since it would always link them to the default project.